rep-2010.rst

```diff
@@ -234,7 +234,9 @@ Any existing tooling for aiding users in setting up ROS security should consider
 Command line interface
 ----------------------
 
-TODO: Consider proposing an interface for a CLI here. Although it seems a bit tangent to the core feature, I think it is important to the proposal since it makes it so people can immediately start using the feature.
+The existing ``ros2 service`` tool can be extended using an ``echo`` keyword to monitor service events.
+Internally, it would subscribe to the hidden topics and echo them. The existing command line parameters for topics can be extended to be used with this ``echo`` verb, along with new
+arguments on to filter message content and analyse delays.
```

Comment:
style: one sentence per line
```suggestion
Internally, it would subscribe to the hidden topics and echo them.
The existing command line parameters for topics can be extended to be used with this ``echo`` verb, along with new
arguments on to filter message content and analyse delays.
```